packages/factory_reset_tools/lib/pages/creation_progress_page.dart

```diff
@@ -8,6 +8,19 @@ import 'package:ubuntu_wizard/ubuntu_wizard.dart';
 import 'package:yaru/foundation.dart';
 import 'package:yaru/widgets.dart';
 
+String displayCreationStatusName(
```

Comment:
This can be done as an `extension` of the enum instead, then none of the other code has to change.

Reply:
Thanks, I followed your recommendation to use `extension on` instead of a standalone function.